src/http.c

```diff
@@ -280,17 +280,20 @@ static int do_http_request(struct tunnel *tunnel,
 	int ret;
 	const char *template = ("%s %s HTTP/1.1\r\n"
 	                        "Host: %s:%d\r\n"
-	                        "User-Agent: Mozilla/5.0 SV1\r\n"
-	                        "Accept: text/plain\r\n"
-	                        "Accept-Encoding: identity\r\n"
+	                        "User-Agent: %s\r\n"
+	                        "Accept: */*\r\n"
+	                        "Accept-Encoding: gzip, deflate, br\r\n"
+	                        "Pragma: no-cache\r\n"
+	                        "Cache-Control: no-store, no-cache, must-revalidate\r\n"
+	                        "If-Modified-Since: Sat, 1 Jan 2000 00:00:00 GMT\r\n"
```

Comment:
These came from both Windows and Mac clients (both emitted the same with the exception of the user agent. I haven't tested any other way, but would imagine that all of these would be ignored since they relate to caching of content and not relevant here.

Reply:
OK, let's keep these changes.
Since both Mac and Windows clients emit these lines we should probably do the same.